packages/replay/rollup.npm.config.js

```diff
@@ -23,12 +21,7 @@ export default makeNPMConfigVariants(
         // set exports to 'named' or 'auto' so that rollup doesn't warn about
         // the default export in `worker/worker.js`
         exports: 'named',
-
-        // As we are inlining the rrweb dependency here, we need to ensure the nested folders are correct
-        // Without this config, you get:
-        // * build/npm/esm/node_modules/rrweb/...
-        // * build/npm/esm/packages/replay/...
-        preserveModulesRoot: path.join(process.cwd(), 'src'),
+        preserveModules: false,
```

Comment:
While this is the default value for rollup configs; we need to set it explicitly here because `makeBaseNPMConfig` sets it to `true`.